Maestro/subscriptions.json

```diff
@@ -1281,6 +1281,40 @@
           "CommitsToMirror": "<trigger-commitlist>"
         }
       }
+    },
+    // Automate opening PRs to merge winforms-designer release/16.10p4 to release/16.11p1
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/winforms-designer/blob/release/16.10p4/**/*"
+      ],
+      "action": "github-dnceng-branch-merge-pr-generator",
+      "actionArguments": {
+        "vsoSourceBranch": "main",
+        "vsoBuildParameters": {
+          "GithubRepoOwner": "dotnet",
+          "GithubRepoName": "<trigger-repo>",
+          "HeadBranch": "<trigger-branch>",
+          "BaseBranch": "release/16.11p1",
+          "ExtraSwitches": "-QuietComments"
+        }
+      }
+    },
+    // Automate opening PRs to merge winforms-designer release/16.11xx to main
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/winforms-designer/blob/release/16.11xx/**/*"
```

Comment:
To get what you want here, eliminate the `xx`.

Reply:
Done. Thank you.